src/test/java/com/csc530/smtlib/SmtLibRunnerTest.java

```diff
@@ -0,0 +1,109 @@
+package com.csc530.smtlib;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import org.junit.Test;
+
+public class SmtLibRunnerTest {
+
+    private static final String SATISFIABLE = "sat";
+    private static final String UNSATISFIABLE = "unsat";
+
+    @Test
+    public void testBooleanDeclaration() {
+        SmtLibRunner runner = new SmtLibRunner();
```

Comment:
@Before method to setup this runner object

change all methods to throw exceptions so you don't have to wrap these in a try catch